test/benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Simeon Ehrig
+#
+# This file is part of vikunja.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+cmake_minimum_required(VERSION 3.18)
+
+add_library(vikunjaBenchSetup INTERFACE)
+target_compile_definitions(vikunjaBenchSetup INTERFACE CATCH_CONFIG_ENABLE_BENCHMARKING)
+target_include_directories(vikunjaBenchSetup INTERFACE include)
+add_library(vikunja::benchSetup ALIAS vikunjaBenchSetup)
+
+target_compile_definitions(vikunjaTestSetup PRIVATE CATCH_CONFIG_ENABLE_BENCHMARKING)
+
+if(VIKUNJA_ENABLE_CUDA_THRUST_BENCHMARKS)
```

Comment:
`cmake_dependent_option` can be dependent on multiple other options (see [here](https://cmake.org/cmake/help/v3.18/module/CMakeDependentOption.html)). Maybe it is possible to let `VIKUNJA_ENABLE_CUDA_THRUST_BENCHMARKS` depend on `ALPAKA_ACC_GPU_CUDA_ENABLE`?